piksi_tools/console/utils.py

```diff
@@ -237,6 +237,15 @@ def datetime_2_str(datetm):
     return (datetm.strftime('%Y-%m-%d %H:%M'), datetm.strftime('%S.%f'))
 
 
+# Modified based on https://stackoverflow.com/a/1094933
+def sizeof_fmt(num, suffix='B'):
+    for unit in ['', 'K', 'M', 'G', 'T', 'P', 'E', 'Z']:
+        if abs(num) < 1024.0:
+            return "%3.1f%s%s" % (num, unit, suffix)
```

Comment:
nit: i prefer format strings https://pyformat.info/#simple

Reply:
good catch and I agree. I will update it